xchange-okcoin/src/main/java/com/xeiam/xchange/okcoin/dto/trade/OkCoinOrdersResult.java

```diff
@@ -2,11 +2,11 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 
-public class OkCoinOrderResult extends OkCoinErrorResult {
+public class OkCoinOrdersResult extends OkCoinErrorResult {
```

Comment:
I disagree, this class is only used inside the library, this is DTO class and not the API class